x-pack/plugins/observability/public/pages/slo_details/components/header_control.tsx

```diff
@@ -52,6 +52,14 @@ export function HeaderControl({ isLoading, slo }: Props) {
     setRuleFlyoutVisibility(true);
   };
 
+  const handleNavigateToRules = () => {
+    navigateToUrl(
+      basePath.prepend(
+        `${paths.observability.rules}?_a=(lastResponse:!(),search:%27%27,params:(sloId:%27${slo?.id}%27),status:!(),type:!())`
```

Comment:
Why don't you create a locator for URL creation? Similar to what we have for `rule details` page?

Reply:
This is simpler :)

Reply:
Indeed it is simpler, but it is not according to best practices and seems fragile.
Shall we have a ticket for it?

Reply:
[Best practices for navigation inside Kibana](https://docs.elastic.dev/kibana-dev-docs/routing-and-navigation)

> - Avoid hardcoding other app's URL in your app's code.
> - Avoid generating other app's state and serializing it into URL query params.
>
> Instead, each app should expose [a locator](https://github.com/elastic/kibana/blob/main/src/plugins/share/common/url_service/locators/README.md). Other apps should use those locators for navigation or URL creation.

Here is a ticket for it: https://github.com/elastic/kibana/issues/155685